### ISSUE TITLE
RenderTransformOverfide does not apply in some situations

### ISSUE THREAD

**Issue by sjsanjsrh** [Yellow-Dog-Man/Resonite-Issues#1582]
### Describe the bug?

Context should have been UserView, but it only applies to ActiveUser in that slot

### To Reproduce

resrec:///U-Sinduy/R-7c991cf3-7455-410d-a5fb-0b49c944b264

### Expected behavior

Handle that processing only for ActiveUser.

### Screenshots

![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/16241081/b82166cf-e95e-4f6f-a876-0bbed27e0a9d)

### Resonite Version Number

2024.3.12.1169

### What Platforms does this occur on?

Windows

### Log Files

[DESKTOP-J80AA7K_-_2024.3.12.1169_-_2024-04-03_00_09_18.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/14839375/DESKTOP-J80AA7K_-_2024.3.12.1169_-_2024-04-03_00_09_18.log)

### Reporters

Sinduy

**Comment by JackTheFoxOtter** [comment]
This is somewhat expected, UserView basically means ActiveUserView. That boils down to how the component is used 95% of the time, which is hiding pieces of your avatar only for your first person view. So I'm pretty sure this isn't a bug but intended behavior, however, it would be useful to be able to specify "only active user" separately from the render context in some situations.

**Comment by JackTheFoxOtter** [comment]
There is no "opposite" render context, that's the underlying problem I think. Being able to make "only active user" a separately togglable property regardless of render context would allow a bunch of configurations that currently aren't easily possible, since UserView only works for the active users (and the other render contexts can't be limited to only one user). This is a limitation I've also run into very recently working on some visual effects.

**Comment by JackTheFoxOtter** [comment]
> Can you give more detail on what you are trying to accomplish specifically, @JackTheFoxOtter? Either way- that sounds like an enhancement/feature request and not a bug, and should probably be its own issue.

Yeah, that's why I said in the beginning I don't think this is a bug, but there is a potential feature request here. My problem was the same, I wanted to create a overwrite for everyone's view context (but ONLY the view context), and wasn't able to, because the UserView is always exclusive for the active user only.

**Comment by JackTheFoxOtter** [comment]
> Could you not get the same result then by setting its desired appearance as the default, and then adding overrides for every other view type, @JackTheFoxOtter?
> 
> If you want to create an override for everyone else's view- that should probably just be the default state of the object?

Yes, but then instead of one matrix recalculation, I do multiple, right? Or are multiple render transform overwrites "free" when they are on the same mesh? I might be misunderstanding how this componenent functions, but my assumption is that there already is a check in there that checks if the local user is the active user before overwriting the UserView. Making that togglable for all render contexts sounds like an easy tweak to get a bunch more flexibility out of that component (unless I'm looking over something here, which might very well be the case!). It would definitely make the behavior more consistent with the other render contexts.
